libs/sdk-ui-dashboard/src/drill/hooks/useDrill.ts

```diff
@@ -0,0 +1,22 @@
+// (C) 2020-2021 GoodData Corporation
+
+import { useDashboardCommandProcessing } from "../../dashboard/useDashboardCommandProcessing";
+import { DashboardDrillTriggered, DashboardCommandFailed, drill } from "../../model";
+
+/**
+ * @internal
+ */
+export const useDrill = (
+    onSuccess?: (event: DashboardDrillTriggered) => void,
```

Comment:
We should probably make this a config object, like in other hooks. This would allow us to specify any combination of the callbacks.